sdk/documentintelligence/ai-document-intelligence-rest/src/documentIntelligence.ts

```diff
@@ -63,5 +65,39 @@ export default function createClient(
     },
   });
 
+  client.pipeline.addPolicy({
+    name: "ResponseStreamToUint8ArrayPolicy",
+    sendRequest: async (req, next) => {
+      logger.info("ResponseStreamToUint8ArrayPolicy starts");
+
+      const figuresUrlPattern = /\/documentModels\/[^/]+\/analyzeResults\/[^/]+\/figures\/[^/]+/;
+      if (req.method === "GET" && figuresUrlPattern.test(req.url)) {
+        logger.info(`Matched URL pattern for figures: ${req.url}`);
+        req.streamResponseStatusCodes = new Set([200]);
+        logger.info("Set streamResponseStatusCodes to 200 for the figures API");
+      }
+
+      const response = await next(req);
+
+      if (req.method === "GET" && figuresUrlPattern.test(req.url)) {
+        logger.info(`Handling response for figures URL: ${req.url}`);
+        if (!response.readableStreamBody) {
+          logger.error("Expected response to have a readable stream but found none");
+          throw new RestError("Expected response to have a readable stream", {
```

Comment:
can also attach the original request here

Reply:
attached req, response and status